src/gc-stock.h

```diff
@@ -580,6 +511,7 @@ extern uv_cond_t gc_threads_cond;
 extern uv_sem_t gc_sweep_assists_needed;
 extern _Atomic(int) gc_n_threads_marking;
 extern _Atomic(int) gc_n_threads_sweeping;
+extern _Atomic(int) nrunning;
```

Comment:
This is not a good name for a global that is externally visible.

Reply:
Refactored.